language/move-core/types/src/u256.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) The Move Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use std::fmt;
+
+use primitive_types::U256;
+use serde::{Deserialize, Serialize};
+
+#[allow(non_camel_case_types)]
+#[derive(Clone, Deserialize, Serialize, Debug, PartialEq, Eq, Hash)]
+pub struct u256(U256);
+
+impl fmt::Display for u256 {
+    fn fmt(&self, f: &mut fmt::Formatter) -> std::fmt::Result {
+        write!(f, "{}", self.0)
```

Comment:
Probably want to delegate for any special cases (around hex and what not)
So `fmt::Display(self.0, f)` or I think `self.0.fmt(f)`

Reply:
Agreed. I haven't spent enough time fleshing this out yet. I reckon there'll be lost of things to impl in subsequent PRs.
For now I'm focusing on the bare minimum

Reply:
I just mean that this specific case for Display can be simpler, while being more robust

Reply:
But understood that more impls will come later

Reply:
> I just mean that this specific case for Display can be simpler, while being more robust
Got it. Will address